catalog/USGS/NLCD_RELEASES/USGS_RCMAP_RELEASES_V5_COMPONENTS.jsonnet

```diff
@@ -0,0 +1,358 @@
+local id = 'USGS/NLCD_RELEASES/USGS_RCMAP_RELEASES_V5_COMPONENTS';
+local subdir = 'USGS/NLCD_RELEASES';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+local parent_url = catalog_subdir_url + 'catalog.json';
+local self_url = catalog_subdir_url + base_filename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'Rangeland Condition Monitoring Assessment and Projection (RCMAP) Fractional Component Time-Series Across the Western U.S. 1985-2021',
+  version: 'v5',
```

Comment:
this is a replacement for https://developers.google.com/earth-engine/datasets/catalog/USGS_NLCD_RELEASES_2019_REL_RCMAP_V4_COVER , right? Should the title be the same?

Also, could you please add versioning (successor / predecessor / latest) links to both v4 and v5 .  See the two CAS PML files here for an example: https://github.com/google/earthengine-catalog/tree/main/catalog/CAS

Reply:
I did base it off of that file yes. Should the new one be renamed to: USGS_NLCD_RELEASES_2021_REL_RCMAP_V5_COVER?
2021 is the latest year available with this release, and it is V5. 
I'm not sure what you mean by versioning links. Do you mean in the Title? They do have V4 and V5 in the Titles, and it is identified on line 26.

Reply:
Changed to USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_COVER